recipes/slidingwindow/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "slidingwindow" %}
+{% set version = "0.0.14" %}
+{% set sha256 = "4c52adf24e2dd6786b9c1c49144083254d9bd9de655b4f2f17bd78cc41a01e49" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script:
+    - export SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}    # [unix]
```

Comment:
do you really need this line?
I didn't see any reference to ``setuptools_scm`` in this project, but I might be missing something
```suggestion
```